resources/quality/ultimaker_s5/um_s5_aa0.8_PLA_Verydraft_Print.inst.cfg

```diff
@@ -17,26 +17,28 @@ cool_fan_speed_max = =100
 cool_min_speed = 2
 gradual_infill_step_height = =3 * layer_height
 infill_line_width = =round(line_width * 0.65 / 0.75, 2)
-infill_pattern = cubic
+infill_pattern = triangles
+layer_height = 0.3
```

Comment:
This is also in the global profile: https://github.com/Ultimaker/Cura/blob/4a26cf0cbdba9b40841b3154a2f7775c8a5635b0/resources/quality/ultimaker_s5/um_s5_global_Verydraft_Quality.inst.cfg#L14
Don't add it here.